backend/utils/data.py

```diff
@@ -3,122 +3,116 @@
 # %%
 import pandas as pd
 import numpy as np
```

Comment:
It doesn't look like numpy is used at all - might as well remove for now?

Reply:
Removed